src/plugins/LADTable/components/LADRow.vue

```diff
@@ -64,14 +64,15 @@ export default {
     },
     computed: {
         formattedTimestamp() {
-            return this.timestamp !== undefined ? this.formats[this.timestampKey].format(this.timestamp) : '---';
+            return this.timestamp !== undefined ? this.getFormattedTimestamp(this.timestamp) : '---';
         }
     },
     mounted() {
         this.metadata = this.openmct.telemetry.getMetadata(this.domainObject);
         this.formats = this.openmct.telemetry.getFormatMap(this.metadata);
         this.keyString = this.openmct.objects.makeKeyString(this.domainObject.identifier);
         this.bounds = this.openmct.time.bounds();
+        console.log('metadata, formats', this.metadata, this.formats);
```

Comment:
Looks like some debug output has been left in?

Reply:
Removed